### PR TITLE
Feature/support gnome 3 32

### PR DESCRIPTION
**Pull request by quarksplitter**
Disclaimer: I've never done any Gnome Extension development before. But I love this one and I've couldn't live without it on Gnome 3.32 so I've decided to give it a try and update it to work with the latest version of Gnome.

If you see any nonsense please point it out. 

Also sorry for a lot of indentations changes.

### CODE REVIEW

**Comment by jonafato** [comment]
@gpouilloux It'd be great to get a new release with these fixes. Anything we can do to help?

**Comment by gpouilloux** [comment]
@quarksplitter I'm concerned about possible breaking changes with older Gnome Shell versions. I use Gnome Shell 3.26.
I will comment your pull request with warnings/errors I had on my system.

File: src/dockerMenu.js [hunk 1]
```diff
@@ -29,70 +29,73 @@ const Docker = Me.imports.src.docker;
 const DockerSubMenuMenuItem = Me.imports.src.dockerSubMenuMenuItem;
 
 // Docker icon on status menu
-const DockerMenu = new Lang.Class({
-    Name: 'DockerMenu.DockerMenu',
-    Extends: PanelMenu.Button,
-
+const DockerMenu = GObject.registerClass(
```
Comment (gpouilloux) [inline src/dockerMenu.js]: I am not familiar at all with the construct `GObject.registerClass` but it raises an error.
```
Extension "docker_status@gpouilloux" had error: TypeError: GObject.registerClass() used with invalid base class (is PanelMenuButton)
```
I'd recommend using the same construct as for the `DockerMenuItem` class.

Comment (gpouilloux) [inline src/dockerMenu.js]: I am unsure if this warning was raised before your PR but Gnome Shell recommends using `var` over `let` or `const` for exported variables.
See similar issue: https://github.com/workspace-grid/workspace-grid/issues/56

File: src/dockerMenu.js [hunk 2]
```diff
@@ -29,70 +29,73 @@ const Docker = Me.imports.src.docker;
...
     // Init the docker menu
-    _init: function () {
-        this.parent(0.0, _("Docker containers"));
+    _init() {
+      super._init(0.0, _("Docker containers"));
 
-        const hbox = new St.BoxLayout({ style_class: 'panel-status-menu-box' });
-        const gicon = Gio.icon_new_for_string(Me.path + "/docker.svg");
-        const dockerIcon = new St.Icon({ gicon: gicon, icon_size: '24' });
+      const hbox = new St.BoxLayout({ style_class: "panel-status-menu-box" });
```
Comment (gpouilloux) [inline src/dockerMenu.js]: I prefer keeping single quotes if it is valid for Gnome Shell 3.32

File: src/dockerMenu.js [hunk 3]
```diff
@@ -29,70 +29,73 @@ const Docker = Me.imports.src.docker;
...
+      const gicon = Gio.icon_new_for_string(Me.path + "/docker.svg");
+      const dockerIcon = new St.Icon({ gicon: gicon, icon_size: "24" });
 
-        hbox.add_child(dockerIcon);
```
Comment (gpouilloux) [inline src/dockerMenu.js]: Can you fix all the indentation issues?

**Comment by gpouilloux** [comment]
@quarksplitter I made the necessary changes from a branch I checkout from your contribution. (See https://github.com/gpouilloux/gnome-shell-extension-docker/pull/42)
Thanks again for your help on this :+1:

**Pull request closed without merging**